src/NHibernate.Validator/Cfg/MappingSchema/Validator.GeneratedSchema.cs

```diff
@@ -1,651 +1,671 @@
+	[System.Xml.Serialization.XmlRootAttribute("valid", Namespace="urn:nhibernate-validator-1.0", IsNullable=false)]
+	public partial class NhvmValid {
+	}
+	
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("NhvXsd", "0.0.0.0")]
+	[System.SerializableAttribute()]
+	[System.Diagnostics.DebuggerStepThroughAttribute()]
+	[System.ComponentModel.DesignerCategoryAttribute("code")]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true, Namespace="urn:nhibernate-validator-1.0")]
+	[System.Xml.Serialization.XmlRootAttribute("nhv-mapping", Namespace="urn:nhibernate-validator-1.0", IsNullable=false)]
+	public partial class NhvMapping {
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlElementAttribute("class")]
+		public NhvmClass[] @class;
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public string @namespace;
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public string assembly;
+	}
+	
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("NhvXsd", "0.0.0.0")]
+	[System.SerializableAttribute()]
+	[System.Diagnostics.DebuggerStepThroughAttribute()]
+	[System.ComponentModel.DesignerCategoryAttribute("code")]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true, Namespace="urn:nhibernate-validator-1.0")]
+	[System.Xml.Serialization.XmlRootAttribute("class", Namespace="urn:nhibernate-validator-1.0", IsNullable=false)]
+	public partial class NhvmClass {
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlElementAttribute("attributename")]
+		public NhvmClassAttributename[] attributename;
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlElementAttribute("property")]
+		public NhvmProperty[] property;
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public string name;
+	}
+	
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("NhvXsd", "0.0.0.0")]
+	[System.SerializableAttribute()]
+	[System.Diagnostics.DebuggerStepThroughAttribute()]
+	[System.ComponentModel.DesignerCategoryAttribute("code")]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true, Namespace="urn:nhibernate-validator-1.0")]
+	public partial class NhvmClassAttributename {
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public string message;
+		
+		/// <remarks/>
+		[System.Xml.Serialization.XmlTextAttribute()]
+		public string[] Text;
+	}
+
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("NhvXsd", "0.0.0.0")]
+	[System.SerializableAttribute()]
+	[System.Diagnostics.DebuggerStepThroughAttribute()]
+	[System.ComponentModel.DesignerCategoryAttribute("code")]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true, Namespace = "urn:nhibernate-validator-1.0")]
+	[System.Xml.Serialization.XmlRootAttribute("enum", Namespace = "urn:nhibernate-validator-1.0", IsNullable = false)]
+	public partial class NhvmEnum

```

Comment:
It appears this had been manually added. This file is generated from `nhv-mapping.xsd` and should be manually edited. It should be re-generated from the schema with the help of the `NHibernate.Tool.HbmXsd` project of NHibernate.Validator solution. Done in #14.